test/supplemental_genbank_tests/genbank_upload_full_test.py

```diff
@@ -1,14 +1,26 @@
+import json
 import os
+import pytest
+import re
 import shutil
 import time
 import unittest
 from configparser import ConfigParser
+from datetime import datetime
 
 from installed_clients.DataFileUtilClient import DataFileUtil
 from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
 from GenomeFileUtil.GenomeFileUtilServer import MethodContext
 from installed_clients.WorkspaceClient import Workspace as workspaceService
+from conftest import assert_exception_correct
 
+_GENOMEFILEUTIL_VERSION = "local-dev"
+_DATAFILEUTIL_VERSION = "0.2.1-release"
+_ASSEMBLYUTIL_VERSION = "3.1.1-release"
+_GENOMEANNOTATIONAPI_VERSION = "1.0.2-release"
+_WSLARGEDATAIO_VERSION = "0.0.5-beta"
```

Comment:
For these four, rather than hardcoding them and having to change them all the time, I'd just pull them out of the provenance and check that they match a regex like `^\d+\.\d+\.\d+-(release|beta)$`

Didn't test that btw

Reply:
👍

Reply:
:+1: